.F
======================================================================
FAIL: test_youdao (test_youdao.test_youdao)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\mars\workspace\git\autotest\src\com\test\case\test_case\test_youdao.py", line 17, in test_youdao
    self.assertEqual(title, "selenium -有道搜索")
AssertionError: 'selenium - 有道搜索' != 'selenium -有道搜索'
- selenium - 有道搜索
?           -
+ selenium -有道搜索


----------------------------------------------------------------------
Ran 2 tests in 31.963s

FAILED (failures=1)
..
----------------------------------------------------------------------
Ran 2 tests in 64.303s

OK
